<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FB-schaudhary-ic" name="malive.InformationalRun" tests="9" failures="0" timestamp="19 May 2017 10:00:24 GMT" time="79.990" errors="1">
  <testcase name="dshboardView" time="50.932" classname="malive.InformationalRun">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at malive.InformationalMethod.L1_dashboardView(InformationalMethod.java:147)
	at malive.InformationalRun.dshboardView(InformationalRun.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- dshboardView -->
  <testcase name="subscriptionCriteria" time="0.191" classname="malive.InformationalRun"/>
  <testcase name="storeSlection" time="3.003" classname="malive.InformationalRun"/>
  <testcase name="publishing" time="0.319" classname="malive.InformationalRun"/>
  <testcase name="campaignCreation" time="5.612" classname="malive.InformationalRun"/>
  <testcase name="defineCmapaign" time="3.070" classname="malive.InformationalRun"/>
  <testcase name="definingResponse" time="8.948" classname="malive.InformationalRun"/>
  <testcase name="scheduling" time="5.757" classname="malive.InformationalRun"/>
  <testcase name="campaignNaivation" time="2.158" classname="malive.InformationalRun"/>
</testsuite> <!-- malive.InformationalRun -->
